dds/DCPS/DataReaderImpl.h

```diff
@@ -585,6 +585,10 @@ class OpenDDS_Dcps_Export DataReaderImpl
   void return_handle(DDS::InstanceHandle_t handle);
 
 protected:
+
+  // Perform cast to get extended version of listener (otherwise nil)
+  virtual DataReaderListener_ptr get_ext_listener();
```

Comment:
Does it need `virtual`?

Reply:
Ah, no, just copied from get_listener(). I'll remove them.